tools/eslint-config-web3-base/ts-jest.js

```diff
@@ -0,0 +1,31 @@
+module.exports = {
+	parser: '@typescript-eslint/parser',
+	plugins: ['@typescript-eslint', 'jest'],
+	extends: ['./ts', 'plugin:jest/recommended', 'plugin:jest/style'],
+	env: {
+		'jest/globals': true,
+	},
+	rules: {
+		'jest/valid-title': ['error'],
+		'jest/no-conditional-expect': ['error'],
+		'import/no-extraneous-dependencies': ['error', { devDependencies: true }],
+		'jest/consistent-test-it': ['error'],
+		'class-methods-use-this': ['error'],
+		'@typescript-eslint/no-magic-numbers': 'off',
```

Comment:
I think this is a good rule to keep enabled

Reply:
In tests we got a lot of cases to write expectations with numbers e.g. 

```js
expect(obj.value).toBeGreaterThan(10); 
```

This rule makes it difficult to write such expectations and its inconvenient for tests to always declare constants for such values and then write expectations.

Reply:
Even in these cases I prefer to document what the number represents by giving is a constant/variable name. Additionally, it makes debugging/updating values easier when all test values are declared at the top of the test/test suite

Reply:
I don't think its convenient for writing tests. Simple statements like `jest.setTimeout(5000);`  would require to declare a constant and use it. But I am changing it anyway, we can turn it off as soon we encounter that case.